web/src/.eslintrc.js

```diff
@@ -23,6 +32,15 @@ module.exports = {
     // handled by prettier
     'react/jsx-curly-newline': 'off',
     'react/jsx-indent': 'off',
+
+    // typescript-eslint rules
+    '@typescript-eslint/explicit-function-return-type': 'off',
+    '@typescript-eslint/no-namespace': 'off',
+    '@typescript-eslint/no-use-before-define': 'off',
+    '@typescript-eslint/no-explicit-any': 'off',
```

Comment:
how much effort would it take to have this one stay on? we probably shouldn't use `any` anywhere we can help it